azurelinuxagent/ga/policy/policy_engine.py

```diff
@@ -50,7 +44,6 @@ def __init__(self, msg, inner=None):
         msg = "Customer-provided policy file ('{0}') is invalid, please correct the following error: {1}".format(conf.get_policy_file_path(), msg)
         super(InvalidPolicyError, self).__init__(msg, inner)
 
-
```

Comment:
Can we add an INFO message just after the check for enabled stating that we are using Policy? This makes clearer the fact that we are now processing policies.

Reply:
__read_policy() is called right after the check for enabled, and it logs the following statement: 

> Policy enforcement is enabled. Enforcing policy using policy file found at '\<path\>'. File contents: \<policy\>

Is that sufficient, or do you think we need an additional log message?

Reply:
It's sufficient, but the message should probably be in the caller instead of read_policy. Who knows, as code evolves we may add other code before read_policy, or call read_policy multiple times.

Alternatively, the caller can log "Policy enforcement is enabled." and read_policy "Enforcing policy using policy file found at '<path>'. File contents: <policy>"

Reply:
Updated